internal/storage/storage.go

```diff
@@ -306,6 +353,69 @@ func CopyPackages(r *git.Repository, sourceStage, destinationStage string, packa
 	return nil
 }
 
+// CopyOverLocalPackage function updates the local repository with the selected local package.
+// It returns the commit hash for the HEAD.
+//
+// Principle of operation
+// 0. Git index is clean.
+// 1. All files need to be removed from the destination folder (in Git repository).
+// 2. Copy integration content to the destination folder.
```

Comment:
It might not be an integration package.
```suggestion
// 2. Copy package content to the destination folder.
```